railties/lib/rails/tasks/routes.rake

```diff
@@ -19,6 +19,9 @@ task routes: :environment do
 
   OptionParser.new do |opts|
     opts.banner = "Usage: rails routes [options]"
+
+    Rake.application.standard_rake_options.each { |args| opts.on(*args) }
```

Comment:
This is iffy. We'd likely need to swallow rake options for every Rails command with custom option parsing.